app/bundles/WebhookBundle/Form/DataTransformer/EventsToArrayTransformer.php

```diff
@@ -17,7 +20,9 @@ public function __construct(
     /**
      * Convert from the PersistentCollection of Event entities to a simple array.
      *
-     * @return array
+     * @phpstan-param $events Collection<int, Event>
```

Comment:
```suggestion
     * @param $events Collection<int, Event>
```
I don't see a reason for phpstan-param.

Reply:
Correct, sorry, i'm used to have those.